src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/azure/AzureProvider.java

```diff
@@ -695,39 +778,39 @@ public String addToInstancePublicIp(Infrastructure infrastructure, String instan
 
         // If all existing network interfaces already have a public IP, then create a new/secondary network interface
         if (networkInterfaces.stream()
-                             .allMatch(netIf -> Optional.ofNullable(netIf.primaryIpConfiguration()).isPresent() &&
-                                                Optional.ofNullable(netIf.primaryIpConfiguration().getPublicIpAddress())
+                             .allMatch(netIf -> Optional.ofNullable(netIf.primaryIPConfiguration()).isPresent() &&
+                                                Optional.ofNullable(netIf.primaryIPConfiguration().getPublicIPAddress())
                                                         .isPresent())) {
             // Reuse the network configuration (virtual private network & security group) of the primary network interface
-            addPublicIpWithNewSecondaryNetworkInterface(azureService, vm, resourceGroup, publicIpAddress);
+            addPublicIpWithNewSecondaryNetworkInterface(azureService, vm, resourceGroup, PublicIPAddress);
             // Otherwise add the public address to the first network interface without public IP
         } else {
             networkInterfaces.stream()
-                             .filter(netIf -> !Optional.ofNullable(netIf.primaryIpConfiguration()).isPresent() ||
-                                              !Optional.ofNullable(netIf.primaryIpConfiguration().getPublicIpAddress())
+                             .filter(netIf -> !Optional.ofNullable(netIf.primaryIPConfiguration()).isPresent() ||
+                                              !Optional.ofNullable(netIf.primaryIPConfiguration().getPublicIPAddress())
                                                        .isPresent())
                              .findFirst()
                              .ifPresent(pubIpAddr -> pubIpAddr.update()
-                                                              .withExistingPrimaryPublicIpAddress(publicIpAddress)
+                                                              .withExistingPrimaryPublicIPAddress(PublicIPAddress)
                                                               .apply());
         }
 
-        return publicIpAddress.ipAddress();
+        return PublicIPAddress.ipAddress();
     }
 
     protected void addPublicIpWithNewSecondaryNetworkInterface(Azure azureService, VirtualMachine vm,
-            ResourceGroup resourceGroup, PublicIpAddress publicIpAddress) {
+            ResourceGroup resourceGroup, PublicIPAddress PublicIPAddress) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)